internal/replace.go

```diff
@@ -180,3 +180,42 @@ func hasExtraDiv(n *html.Node) bool {
 
 	return false
 }
+
+// TextFormatter catches Bold, Italic or underlined formatting
```

Comment:
Underline is not actually implemented. Not here, not in Markdown specification )
```suggestion
// TextFormatter catches bold and italic 
```